### DISCUSSION TITLE
Detect and remove automated accounts

### DISCUSSION THREAD

**Discussion by BastianZim** [community/community#13134] [Other Feature Feedback, Questions, & Ideas]
Hi,

I like the idea but my feed is currently getting cluttered with automated accounts. They are not marked as a bot outrightly but are one. It would be good if GitHub could detect that and not show them.

In my example it's https://github.com/regro-cf-autotick-bot

**Comment by sol173angs** [comment]
Only remove automated accounts that are distance greater than 1 link from any human user AND  not redundant or spam.

**Comment by EmilStenstrom** [comment]
Hi, currently my feed is filled with content from a bot called [regro-cf-autotick-bot](https://github.com/regro-cf-autotick-bot). I think it has done something conda-related to one of my repos once (so it's 1 link away from me), but I really don't want it. Feed is filled with random conda repos that it automatically interacts with.

Not sure if the solution here is to rerank the feed, or just to let me ignore that user from the feed.

**Comment by jhillyerd** [comment]
I have the same problem with an overactive bot, and the `I'm not interested in merged pull requests from ...` option doesn't seem to do anything, even days later.

**Comment by billythekid** [comment]
Hi folks 👋 

If you're experiencing unsolicited or otherwise spammy contact be it via PRs or issue comments or whatever, please do make use of the "Report or block" button on the account's profile page and our teams in charge of this area can check it out for you.

For more information, read [Reporting abuse or spam](https://docs.github.com/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam) in the GitHub Docs.

Cheers,
Billy 🤠

**Reply by BastianZim** [reply]
Hi @billythekid I think there is some misunderstanding. This is about the new beta "For you Beta" feed where Github shows you different things, it is not about people posting something.

**Reply by billythekid** [reply]
Ahh, gotcha! Thanks for clarifying @BastianZim. 

Forgot to check what category we were in there! :bow:

As you were!

![](https://media.giphy.com/media/hLsCCNF5wlxuE99mxV/giphy-downsized-large.gif)
